src/Altinn.Broker.API/Mappers/FileTransferStatusDetailsExtMapper.cs

```diff
@@ -11,8 +11,9 @@ internal static FileTransferStatusDetailsExt MapToExternalModel(FileTransferEnti
         var fileTransferOverview = FileTransferStatusOverviewExtMapper.MapToExternalModel(fileTransfer);
         return new FileTransferStatusDetailsExt()
         {
-            Checksum = fileTransferOverview.Checksum,
             FileTransferId = fileTransfer.FileTransferId,
+            ResourceId = fileTransfer.ResourceId,
+            Checksum = fileTransferOverview.Checksum,
             FileName = fileTransferOverview.FileName,
             Sender = fileTransferOverview.Sender,
             FileTransferStatus = fileTransferOverview.FileTransferStatus,
```

Comment:
Should we add information about virusscan here, if sender is making the request?

Reply:
For sure, fixed.